MapboxNavigation/Resources/ja.lproj/Localizable.strings

```diff
@@ -131,7 +116,7 @@
 "RESUME" = "再開する";
 
 /* The text of a banner that appears during turn-by-turn navigation when route simulation is enabled. */
-"USER_IN_SIMULATION_MODE" = "%dにてナビのシュミレーションを行う";
+"USER_IN_SIMULATION_MODE" = "Simulating Navigation at %@×";
```

Comment:
`USER_IN_SIMULATION_MODE` got fuzzied for every language because `%d` was replaced by `%@`. It should be safe to go through Transifex restoring this message’s translations.